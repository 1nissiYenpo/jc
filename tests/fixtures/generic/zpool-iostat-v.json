[{"pool":"zhgstera6","cap_alloc":2.89,"cap_free":2.2,"ops_read":0,"ops_write":2,"bw_read":349.0,"bw_write":448.0,"cap_alloc_unit":"T","cap_free_unit":"T","bw_read_unit":"K","bw_write_unit":"K"},{"pool":"726060ALE614-K8JAPRGN:10","parent":"zhgstera6","cap_alloc":2.89,"cap_free":2.2,"ops_read":0,"ops_write":2,"bw_read":349.0,"bw_write":448.0,"cap_alloc_unit":"T","cap_free_unit":"T","bw_read_unit":"K","bw_write_unit":"K"},{"pool":"zint500","cap_alloc":230.0,"cap_free":24.0,"ops_read":0,"ops_write":9,"bw_read":112.0,"bw_write":318.0,"cap_alloc_unit":"G","cap_free_unit":"G","bw_read_unit":"K","bw_write_unit":"K"},{"pool":"ST3500418AS-5VMSTSSX:5","parent":"zint500","cap_alloc":230.0,"cap_free":24.0,"ops_read":0,"ops_write":9,"bw_read":112.0,"bw_write":318.0,"cap_alloc_unit":"G","cap_free_unit":"G","bw_read_unit":"K","bw_write_unit":"K"},{"pool":"zsam53","cap_alloc":211.0,"cap_free":24.5,"ops_read":0,"ops_write":0,"bw_read":14.7,"bw_write":74.4,"cap_alloc_unit":"G","cap_free_unit":"G","bw_read_unit":"K","bw_write_unit":"K"},{"pool":"Portable_SSD_T5-S49WNP0N120517B:8","parent":"zsam53","cap_alloc":211.0,"cap_free":24.5,"ops_read":0,"ops_write":0,"bw_read":14.7,"bw_write":74.4,"cap_alloc_unit":"G","cap_free_unit":"G","bw_read_unit":"K","bw_write_unit":"K"}]
