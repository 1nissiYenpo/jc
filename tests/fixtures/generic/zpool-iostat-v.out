                                       capacity     operations     bandwidth
pool                                 alloc   free   read  write   read  write
-----------------------------------  -----  -----  -----  -----  -----  -----
zhgstera6                            2.89T  2.20T      0      2   349K   448K
  726060ALE614-K8JAPRGN:10           2.89T  2.20T      0      2   349K   448K
-----------------------------------  -----  -----  -----  -----  -----  -----
zint500                               230G  24.0G      0      9   112K   318K
  ST3500418AS-5VMSTSSX:5              230G  24.0G      0      9   112K   318K
-----------------------------------  -----  -----  -----  -----  -----  -----
zsam53                                211G  24.5G      0      0  14.7K  74.4K
  Portable_SSD_T5-S49WNP0N120517B:8   211G  24.5G      0      0  14.7K  74.4K
-----------------------------------  -----  -----  -----  -----  -----  -----
